contracts/DelayView.sol

```diff
@@ -5,24 +5,37 @@ import "./DelayManageable.sol";
 
 /// @title DelayView - getter view functions
 contract DelayView is DelayManageable {
-    function GetAllUsersData(address _token)
-        external
-        view
-        returns (address[] memory, Vault[] memory _vaults)
-    {
-        _vaults = new Vault[](Users[_token].length);
-        for (uint256 i = 0; i < Users[_token].length; i++) {
```

Comment:
do we have a getter for 
`Users[_token].length`

Reply:
not yet, we'll need one